pages/common/terraform-fmt.md

```diff
@@ -0,0 +1,20 @@
+# terraform fmt
+
+> Format configuration according to Terraform language style conventions.
+> More information: <https://www.terraform.io/docs/commands/fmt.html>.
+
+- Format configuration in current directory:
```

Comment:
Perhaps this will read better?
```suggestion
- Format the configuration in the current directory:
```

Is it a configuration file? or multiple files? It would help to word it better if we know the nature of the configuration file.

Reply:
It could be a single file or multiple files.